.circleci/config.yml

```diff
@@ -48,6 +48,26 @@ jobs:
 
       - run: npm run lint
 
+  test-integration:
+    docker:
+      - image: circleci/node:14.11.0-stretch
+      # Integration tests need MongoDB server running and accessible on port 27017
+      - image: circleci/mongo:4.0.0
+        command: mongod --oplogSize 128 --replSet rs0 --storageEngine=wiredTiger
+        ports:
+          - "27017:27017"
+    steps:
+      - checkout
+      - run:
+          name: Run Integration Tests
+          environment:
+            MONGO_URL: mongodb://localhost:27017/test
+            MONGO_USE_UNIFIED_TOPOLOGY: false
+          command: |
+            sudo apt-get install -y mongodb
+            mongo --eval "rs.initiate()"
+            npx --quiet --package @reactioncommerce/ci-scripts@1.12.2 run-integration-tests
```

Comment:
I think this command runs all integration tests, not just the ones in this plugin